packages/dgrid-shim/src/index.ts

```diff
@@ -24,10 +24,9 @@ import * as Selection from "dgrid/Selection";
 import * as ColumnResizer from "dgrid/extensions/ColumnResizer";
 import * as CompoundColumns from "dgrid/extensions/CompoundColumns";
 import * as nlsPagination from "dgrid/extensions/nls/pagination";
-import * as Pagination from "dgrid/extensions/Pagination";
 
 //  Other ---
-import { GridHelper } from "./gridHelper";
+import { GridHelper, Pagination } from "./gridHelper";
```

Comment:
Is Pagination used anywhere? I don't see any new references to it?

Reply:
It is - this replaces the current Pagination with a specifialized one.